slidechain/Pegging.md

```diff
@@ -142,18 +142,26 @@ Exporting funds from TxVM for peg-out to Stellar requires three steps:
 2. Change the temporary account’s signer to a
    [preauthorized transaction](https://www.stellar.org/developers/guides/concepts/multi-sig.html#pre-authorized-transaction)
    as described below;
-3. [Retire](https://github.com/chain/txvm/blob/main/specifications/txvm.md#retire)
-   some imported funds in a transaction where the `retire` instruction is immediately followed by a
-   [log](https://github.com/chain/txvm/blob/main/specifications/txvm.md#log)
-   of this JSON string:
+3. Lock the TxVM funds to be exported in a smart contract,
+   to be unlocked after the peg-out step.
+   If peg-out succeeds,
+   the smart contract
+   [retires](https://github.com/chain/txvm/blob/main/specifications/txvm.md#retire)
+   the locked-up funds.
+   If peg-out fails,
+   the smart contract repays the locked-up funds to the exporter.
+   Along with the funds, the smart contract stores the exporter’s pubkey
+   (in case repayment is needed)
+   and a JSON string of the form:
```

Comment:
This JSON string also now contains the amount to export, the TxVM anchor, and the recipient TxVM public key.

Reply:
Thanks! Fixed.